cid.js

```diff
@@ -29,4 +29,12 @@ function CIDToHash(input) {
     return bytesToHex(hashBytes)
 }
 
-module.exports = { CIDToHash, hashToCID }
+function isCID(input) {
+    return input.startsWith('bafyb') && input.length === 59
```

Comment:
is this working with encrypted references as well?

Reply:
It's not working with encrypted references. They would not work as domain names because they would be too long.